src/pyuvdata/uvdata/mwa_corr_fits.py

```diff
@@ -142,6 +142,12 @@ def read_metafits(
         else:
             obs_freq_center = meta_hdr["FREQCENT"] * 1e6
 
+        # error if fringe stopping is turned on
+        if mwax and meta_hdr["DELAYMOD"] == "FULLTRACK":
+            raise ValueError("Fringe stopped data is not supported")
```

Comment:
Let's make this a NotImplementedError and suggest that users make in issue if they encounter this error, maybe something like:
"This data has had fringe stopping applied. We do not yet have support for fringe stopped data, partly because we have not seen files like this yet. Please file an issue in our GitHub issue log so that we can help: https://github.com/RadioAstronomySoftwareGroup/pyuvdata/issues. Please include a link to the data you're trying to read in your issue."